src/request.js

```diff
@@ -0,0 +1,27 @@
+export default class Request {
+  constructor(gameId) {
+    this.gameId = gameId;
+    this.base = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+  }
+
+  async addScore(name, score) {
```

Comment:
- [ ] As mentioned in the project requirements, and as you're working with JS ES6, please use the arrow functions instead of the `function` keyword. 🙏 😄 

**hint**
```
const addScrore = async (name, score) => {
  // code goes here
}
```

Reply:
This is meant to be a class method, though. Wouldn't turning the method into an arrow function defeat the whole purpose of having a class in the first place?

Reply:
- [ ] Please follow a hint given by the previous reviewer for good use of arrow functions

Reply:
Right but that's my point. I don't want to use it outside of a class. I want to use the class. What exactly is the problem with using classes?

Reply:
Can I at least get a response here? This is so rude.

Reply:
Hi @crgc 
Thank you for your communication 😄

The point of using the arrow functions here in this specific project is to practice it while working with the JS ES6 version. I totally understand your desire of using Classes (I myself like using them a lot) but eventually, you'll get many chances to use them in future (more advanced) projects. 😄 🙏